demo/packaged/sql/00-accelerating-batch-analytics.sql

```diff
@@ -0,0 +1,535 @@
+-- Use Case: Accelerating Batch Analytics (accelerating-batch-analytics)
+--
+-- Speed up batch analytics with an always-on incremental Feldera pipeline.
+--
+-- ## How to Run
+--
+-- * Paste this SQL code in the Feldera WebConsole and hit Play ▶️.
+-- * In the Change Stream tab, select the `q1` view.
+-- * You should see a list of changes for the Pricing Summary report of
+--   `lineitem`s.
+-- * Run the ad-hoc query:
+--   `SELECT * FROM q1`
+--   This will display the output of the query.
+-- * To demonstrate incremental computation, insert a new record using the
+--   following ad-hoc query:
+-- ```
+-- INSERT INTO LINEITEM VALUES (1, 5, 4, 1, 50, 0.80, 0.65, 0.10, 'B', 'C', '1998-09-01', '1998-09-01', '1998-09-01', 'DELIVER IN PERSON', 'TRUCK', 'new record insertion')
+-- ```
+-- * Re-run the `SELECT` query to see the updated results.
```

Comment:
```suggestion
-- * Run the ad-hoc query to display the contents of the q1 view: `SELECT * FROM q1`
-- * To demonstrate incremental computation, insert a new record using the following ad-hoc query:
-- ```
-- INSERT INTO LINEITEM VALUES (1, 5, 4, 1, 50, 0.80, 0.65, 0.10, 'B', 'C', '1998-09-01', '1998-09-01', '1998-09-01', 'DELIVER IN PERSON', 'TRUCK', 'new record insertion')
-- ```
-- * Re-run the `SELECT` query to see the updated results.
```